lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -50,11 +53,7 @@ def to_representation(self, course_overview):
                 }
             },
             'course_image': course_overview.course_image_url,
-            'course_about': reverse(
-                'about_course',
-                kwargs={'course_id': course_id},
-                request=request,
-            ),
+            'course_about': None if not course_about_url else "http:{}".format(course_about_url),
```

Comment:
can we please update the name of the key i.e., "course_about" to make it something like **course_about_url** as this holds the URL.